dashboard/app/views/reports/csp_pd_responses.html.haml

```diff
@@ -2,11 +2,17 @@
   CS Principles PD Responses
 
 - if @responses.size > 0
+  #{link_to 'Download CSV', {:format => :csv}}
+
   - @responses.each do |level_id, level_responses|
     %h3
       CSP PD Responses for Level
       = level_id
     - if level_responses.size > 0
+      - if level_responses.size == @response_limit
+        WARNING: The results are truncated, there are additional records not
```

Comment:
Minor nit: technically, if there are exactly `@response_limit` responses before limiting, this isn't true.